pytorch_lightning/loggers/mlflow.py

```diff
@@ -149,6 +153,16 @@ def experiment(self) -> MlflowClient:
             self.logger.experiment.some_mlflow_function()
 
         """
+
+        if self._initialized:
```

Comment:
should it be possible to track this with experiment_id?
```py
if self.__experiment_id is not None:
    return self._mlflow_client
```
or is there any case where `experiment_id is None and initialized is True` or `experiment_id is not None and initialized is False`?

Reply:
@rohitgr7 What you say is totally true, it could be done with that other variable, however I think it is not very readable for future modifications of the code. I would also like to refract this logger in the future to simplify its implementation.